spec/controllers/wishlists_controller_spec.rb

```diff
@@ -144,6 +216,34 @@
         include_examples "redirects to the 'sign_in' page"
       end
     end
+
+    describe "GET #edit" do
+      context "with valid parameters" do
+        before(:each) { get_edit(id: wishlist_id) }
+
+        include_examples "redirects to the 'sign_in' page"
+      end
+
+      context "with invalid parameters" do
+        before(:each) { post_create(id: -1) }
+
+        include_examples "redirects to the 'sign_in' page"
+      end
```

Comment:
you have duplication here. I would put a context for not logged in user to the previous describe "GET #edit".